src/Resources/Locales/en.json

```diff
@@ -461,16 +461,16 @@
             "tabDescription": "Define widgets that appear in the element's context"
         },
         "behaviorVisualRulesTab": {
-            "tabDescription": "Set rules for applying color to certain conditions—for instance, red to indicate high temperatures, or green for optimal speed.",
+            "tabSubtitle": "Create rules for applying visuals to elements based on twin data.",
```

Comment:
did you make sure to change or delete these keys once before changing the values? THat way it will pick them up and generate for all the languages. By default it will not update translations for keys that already exist.

Reply:
I renamed (removed) all the old keys so the new translations would be generated.